src/domains/main/common/components/BestList.jsx

```
@@ -11,24 +12,41 @@ const BestList = ({ boardName, bestList }) => {
         <div className='text-xl font-bold sm:text-3xl'>{boardName} 베스트</div>
       </div>
       <div className='grid grid-flow-row sm:grid-cols-2 gap-y-4 gap-x-20'>
-        {bestList.map(({ title, boardId, commentCount }, index) => {
+        {bestList.map(({ title, boardId, jobPostingId, commentCount, viewCount }, index) => {
```

ESLint

[error] 'bestList.map' is missing in props validation

(react/prop-types)